packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -266,6 +266,13 @@ interface DatabaseClusterBaseProps {
    */
   readonly storageEncryptionKey?: kms.IKey;
 
+  /**
+   * Credentials for the administrative user
+   *
+   * @default - A username of 'admin' (or 'postgres' for PostgreSQL) and SecretsManager-generated password
+   */
+  readonly credentials?: Credentials;
```

Comment:
Good point, I will put this back

Reply:
Actually, are you sure? both/only `DatabaseClusterFromSnapshotProps` and `DatabaseClusterProps` extend `DatabaseClusterBaseProps` (and both would need `credentials`) so it would be DRYer to move it to the base interface? I don't mind but it would be a bit duplicative.

Reply:
Scratch my last comment, I saw your comment below.